egs/wsj/s5/steps/nnet3/libs/common_train_lib.py

```diff
@@ -205,42 +300,28 @@ def ParseModelConfigVarsFile(var_file):
 
     raise Exception('Error while parsing the file {0}'.format(var_file))
 
+def ParseGenericConfigVarsFile(var_file):
+    variables = {}
+    try:
+        var_file_handle = open(var_file, 'r')
+        for line in var_file_handle:
+            parts = line.split('=')
+            field_name = parts[0].strip()
+            field_value = parts[1].strip()
+            if field_name in ['model_left_context', 'left_context']:
+                variables['model_left_context'] = int(field_value)
+            elif field_name in ['model_right_context', 'right_context']:
+                variables['model_right_context'] = int(field_value)
+            elif field_name == 'num_hidden_layers':
```

Comment:
As we recently decided to either make layer-wise pretraining optional or remove it completely we no longer need the `num_hidden_layers` variable. Please see #1170 for related discussion. Also, remember that with more complex and non-linear structures `num-hidden-layers` does not make sense.